minigalaxy/launcher.py

```diff
@@ -65,6 +66,16 @@ def determine_launcher_type(files):
         launcher_type = "final_resort"
     return launcher_type
 
+def get_exe_cmd_with_var_command(game, exe_cmd):
+    command_list = game.get_info("command").split()
+    var_list = game.get_info("variable").split()
+
+    if not(var_list) or var_list[0] == "env":
+        exe_cmd = var_list + exe_cmd + command_list
+    else:
+        var_list.insert(0, "env")
+        exe_cmd = var_list + exe_cmd + command_list
```

Comment:
This is working solution, but we have same command
```
exe_cmd = var_list + exe_cmd + command_list
```
in both "if" and "else", which is not a good practice. I would rather suggest something like:
```
if var_list:
    if var_list[0] not in ["env"]:
        var_list.insert(0, "env")
exe_cmd = var_list + exe_cmd + command_list
```

Reply:
I don't yet have the good habit of simplifying the code.
The code works, i push later the modification.
Thanks